meta_package/BUILD

```diff
@@ -7,8 +7,8 @@ pkg_tar(
     name = "all-package-data",
     extension = "tar.gz",
     deps = [
```

Comment:
This whole metapacking stuff is no longer needed remove the code, but leave the `BUILD` file be.